docs/.vuepress/components/FrameDoc.vue

```diff
@@ -0,0 +1,55 @@
+<template>
```

Comment:
Could you please run lint for this file to fix indentation and other styling issues.

Reply:
It has been fixed